parser/ss.go

```diff
@@ -217,6 +214,16 @@ func (ss *SSParser) ParseAndInsert(meta map[string]bigquery.Value, testName stri
 				ss.TableName(), "ss", "corrupted content").Inc()
 			return err
 		}
+		if ValidateIP(ss_value["LocalAddress"]) != nil {
```

Comment:
These blocks discard the errors returned by `ValidateIP`. https://github.com/m-lab/etl/blob/38c443d1e354096dc28a2e1aadd7c14ae1d1399f/parser/util.go#L21

Would it be helpful to include this message in logs or in the error message for the new error returned below?